src/main/java/com/webs/user/JwtFilter.java

```diff
@@ -0,0 +1,86 @@
+package com.webs.user;
```

Comment:
I know this is the base service, but it could be useful to determine what a good directory structure would be. Ie, if we're putting user specific data (model, controller, repository) is com.webs.user, maybe this would belong in something like com.webs.filter...but then I'm not sure where Application.java should necessarily be (com.webs?)